hazelcast-jet-all/pom.xml

```diff
@@ -127,10 +90,80 @@
                             <artifactId>org.osgi.core</artifactId>
                             <version>4.2.0</version>
                         </additionalDependency>
-
                     </additionalDependencies>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>${maven.bundle.plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>bundle-manifest</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>bundle</goal>
+                        </goals>
+                        <configuration>
+                            <supportedProjectTypes>jar</supportedProjectTypes>
+                            <instructions>
+                                <Embed-Dependency>hazelcast-jet-core;inline=true</Embed-Dependency>
```

Comment:
is this needed? since the JAR is already shaded, and we don't have hazelcast-jet-core as an OSGi module

Reply:
Yes, it is. We need to do it twice because bnd-tool needs all dependencies for producing the Manifest file.